src/modules/github/actions/github.actions.js

```diff
@@ -1,3 +1,4 @@
+import { setMessage } from "../../messages/actions/messages.actions";
```

Comment:
To rozwiązanie jest dopuszczalne, ale mocno wiąże moduł "github" z "errors" - jeden bez drugiego nie będzie mógł działać.

Można zastanowić się nad innym rozwiązaniem. Możemy przekazać `errorHandler` do `getRepos` czyli:

```
export const getRepos = (userName, errorHandler) => (dispatch, getState) => {
   ...
   .catch(errorHandler); // wcześniej .catch((err) => dispatch(setMessage(err)));
```

Teraz tam gdzie wywołujesz akcję `getRepos`, możesz zrobić coś takiego:
```
function errHandler(err) {
    dispatch(setMessage(err));
}

dispatch(getRepos('devmentor-pl', errHandler)); // w ten sposób nie wiążemy na siłę obu modułów 
```